src/renderer/components/composer/EmojiAndStickerPicker.tsx

```diff
@@ -68,22 +70,46 @@ export const StickerPicker = (props: {
     runtime.openPath(folder)
   }
 
+  const stickerPackNames = Object.keys(stickers)
+
   return (
     <div className='sticker-picker'>
-      <button onClick={onOpenStickerFolder}>Open Sticker Folder</button>
-      <div className='sticker-container'>
-        {Object.keys(stickers).map(stickerPackName => {
-          return (
-            <StickerDiv
-              chatId={chatId}
-              key={stickerPackName}
-              stickerPackName={stickerPackName}
-              stickerPackImages={stickers[stickerPackName]}
-              setShowEmojiPicker={setShowEmojiPicker}
-            />
-          )
-        })}
-      </div>
+      {stickerPackNames.length > 0 ? (
+        <>
+          <div className='sticker-container'>
+            {stickerPackNames.map(name => (
+              <DisplayedStickerPack
+                chatId={chatId}
+                key={name}
+                stickerPackName={name}
+                stickerPackImages={stickers[name]}
+                setShowEmojiPicker={setShowEmojiPicker}
+              />
+            ))}
+          </div>
+          <div className='sticker-actions-container'>
+            <button
```

Comment:
should I add an aria-label for this?

Reply:
I don't think it is required when the content of the button is already the text that would be in the aria-label?